api/internal/repo/repository_actions.py

```diff
@@ -0,0 +1,63 @@
+import logging
```

Comment:
This file was removed at some point and added back in a merge conflict I think, going to remove if we should?